src/game/level.rs

```diff
@@ -328,9 +361,14 @@ impl<'a, 'b: 'a> LevelVisitor for LevelBuilder<'a, 'b> {
     }
 
     fn visit_sky_quad(&mut self, &(ref v1, ref v2): &(Vec2f, Vec2f), (low, high): (f32, f32)) {
-        self.scene.sky_buffer()
-            .push(v1, low).push(v2, low).push(v1, high)
-            .push(v2, low).push(v2, high).push(v1, high);
+        self.scene
+            .sky_buffer()
+            .push(v1, low)
```

Comment:
Could you please group the `push` calls in groups of three (the sky_buffer change is fine)? The reason they're in groups of three is because they correspond to triangles and I find the visual guide useful.